### PR TITLE
Update Turkish localisation

### PR DESCRIPTION
**Pull request by bitigchi**

### CODE REVIEW

File: po/tr.po [hunk 1]
```diff
@@ -1875,17 +1873,16 @@ msgid "Aborting migration because mkdir on `%s' failed: %s"
...
 #: rust/libnewsboat/src/filterparser.rs:63
-#, fuzzy
 msgid "attribute name"
-msgstr "özellik bulunamadı"
+msgstr "öznitelik adı bulunamadı"
 
 #: rust/libnewsboat/src/filterparser.rs:64
 msgid "one of: =~, ==, =, !~, !=, <=, >=, <, >, between, #, !#"
-msgstr ""
+msgstr "şunlardan biri: =~, ==, =, !~, !=, <=, >=, <, >, arasında, #, !#"
```
Comment (der-lyse) [inline po/tr.po:1881]: I reckon the `beween` operator must not be translated, but kept in English.

**Review by der-lyse — CHANGES_REQUESTED** [review]
Disclaimer: I don't speak or understand Turkish. Other than one thing, this looks good to me.

**Review by Minoru — APPROVED** [review]
Thank you! I'll merge this once the CI checks are done.

**Comment by Minoru** [comment]
(I don't think there are two ways to implement the changes that @der-lyse requested, so I'm merging on an assumption that he'd approve this already.)

File: po/tr.po [hunk 2]
```diff
@@ -1069,9 +1070,8 @@ msgstr[0] ""
 msgstr[1] ""
```
Comment (dennisschagt) [inline po/tr.po:1070]: It looks like this message is left untranslated.
Other than that, looks good.

Comment (bitigchi) [reply po/tr.po:1070]: Thanks for noticing! Fixed it in my other PR!

Comment (dennisschagt) [reply po/tr.po:1070]: Looks good, thanks!

**Pull request merged**